test/WSUWordControllerTest.cpp

```diff
@@ -0,0 +1,93 @@
+//
+//  CommandControllerTest.cpp
+//  demo
+//
+//  Created by Erik M. Buck on 3/16/21.
+//  Copyright © 2021 Erik M. Buck. All rights reserved.
+//
+
+#include <gtest/gtest.h>
+#include "StoredString.hpp"
+#include "WSUWordController.hpp"
+#include "InsertCharacterAtIndexCommand.hpp"
+
+TEST(WSUWordController, emptyModel) {
+    WSU::Controller::WSUWordController commandController;
+    EXPECT_EQ("", commandController.getCurrentStoredStringValue());
+}
+
+TEST(WSUWordController, runScript0) {
+    WSU::Controller::WSUWordController commandController;
+    commandController.runScript("{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}");
+    EXPECT_EQ("a", commandController.getCurrentStoredStringValue());
+}
+
+TEST(WSUWordController, runScript1) {
+    WSU::Controller::WSUWordController commandController;
+    commandController.runScript("{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}{\"character\": \"-\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}");
```

Comment:
These scripts would be easier to read if each command was on a separate line, taking advantage of the fact that adjacent string literals are concatenated by the compiler in C++.
```suggestion
    commandController.runScript(
        "{\"character\": \"a\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}"
        "{\"character\": \"-\", \"command\": \"insertCharacterAtIndex\", \"index\": 0}"
    );
```

Reply:
Done